api/internal/core/storage/etcd.go

```diff
@@ -120,6 +126,8 @@ func (s *EtcdV3Storage) List(ctx context.Context, key string) ([]Keypair, error)
 		data := Keypair{
 			Key:   string(resp.Kvs[i].Key),
 			Value: string(resp.Kvs[i].Value),
+			// Mark the init_dir etcd event as skippable if value is init_dir
+			Skipped: string(resp.Kvs[i].Value) == SkippedValueEtcdInitDir,
```

Comment:
I think empty string or `{}` should be skipped too.

Reply:
@nic-chen  yep, there definitely exists various kinds of values that should be skippable for each storage type. This PR is going to explicitly skip the `init_dir` values to fix #1329 . How about adding other more condition checks when we need to deal with them?

Reply:
APISIX may change it to `{}`, so I think we should deal with them in this PR if you have time for it. Thanks.

Reply:
> APISIX may change it to `{}`, so I think we should deal with them in this PR if you have time for it. Thanks.

@nic-chen thanks for the directions. I will set `{}` as skippable soon

Reply:
@nic-chen  PR renewed and it's done. Thanks.